Assets/Scripts/Acantilado.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Acantilado : MonoBehaviour
+{
+
```

Comment:
Remove extra spaces, extra spaces only between methods and between methods and fields